packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.md

```diff
@@ -1,8 +1,12 @@
 ---
 title: "Login Page"
 cssPrefix: "pf-c-login"
+section: "components"
+seperatePages: true
 ---
 ## Simple Login Page
+<a href="SimpleLoginPage" target="_blank">Popout example</a>
+
 ```js
```

Comment:
Change to ```nolive to not render the broken preview.

Reply:
Hi @redallen, when I do that I get:
![image](https://user-images.githubusercontent.com/12733153/56155688-818a6600-5f89-11e9-9a2d-01ea1c930c78.png)

Reply:
Looks good! No more confusing errors trying to render a page there that should really be fullscreen.